src/web_interface/components/jinja_filter.py

```diff
@@ -124,6 +125,7 @@ def _setup_filters(self):
         self._app.jinja_env.filters['fix_cwe'] = fix_cwe
         self._app.jinja_env.filters['vulnerability_class'] = vulnerability_class
         self._app.jinja_env.filters['auth_enabled'] = self.check_auth
+	self._app.jinja_env.filters['hash'] = lambda s: hashlib.md5(s.encode()).hexdigest()
```

Comment:
Please use "helperFunctions.hash.get_md5"
Otherwise you must write a test for that  :wink:

Reply:
By the way: I think it would be better to call the filter 'md5_hash' or something like that.